squall/squall_exception.hpp

```diff
@@ -2,13 +2,14 @@
 #define SQUALL_EXCEPTION_HPP_
 
 #include <stdexcept>
+#include "squall_utility.hpp"
 
 namespace squall {
 
 class squirrel_error : public std::runtime_error {
 public:
     using std::runtime_error::runtime_error;
-    squirrel_error(const std::string& s) : std::runtime_error(s) {}
+    squirrel_error(const string& s) : std::runtime_error(locale_converter::to_std_string(s)) {}
```

Comment:
std::runtime_error has a constructor that takes std::string as argument and inherits it.
squirrel_error should support constructor that takes "squall::string"